spec/features/review_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+feature "review" do
+  context "can start" do
+
+    before(:each) do
+      card = create(:card, :expired)
+    end
+
+    it  "can check card" do
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.